tools/benchmark_tool/openvino/tools/benchmark/main.py

```diff
@@ -80,13 +80,25 @@ def is_flag_set_in_command_line(flag):
             cldnn_config = config[GPU_DEVICE_NAME]['CONFIG_FILE']
             benchmark.add_extension(path_to_cldnn_config=cldnn_config)
 
-        if not args.perf_hint:
-            for device in devices:
-                supported_properties = benchmark.core.get_property(device, 'SUPPORTED_PROPERTIES')
-                if 'PERFORMANCE_HINT' in supported_properties:
-                    logger.warning(f"-hint default value is determined as 'THROUGHPUT' automatically for {device} device" +
-                                    "For more detailed information look at README.")
+        for device in devices:
+            supported_properties = benchmark.core.get_property(device, 'SUPPORTED_PROPERTIES')
+            if 'PERFORMANCE_HINT' in supported_properties:
+                if is_flag_set_in_command_line('hint'):
+                    if args.perf_hint=='throughput':
+                        logger.warning(f"Device {device} performance hint is set to THROUGHPUT.")
+                        args.perf_hint = "throughput"
```

Comment:
Why lower case is used? Looks unaligned with UNDEFINED

Reply:
removed UNDEFINED.

Reply:
this assignment looks redundant (already checked with the ==)